gammagl/models/sp2gcl.py

```diff
@@ -0,0 +1,81 @@
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+from gammagl.layers.conv import GCNConv
+class Encoder(nn.Module):
```

Comment:
I remember there are many `Encoder` in the gammagl. I do not recommend you to place this function here.